src/Cle.Common/TypeSystem/ConstantValue.cs

```diff
@@ -48,13 +54,21 @@ public static ConstantValue SignedInteger(long value)
         }
 
         /// <summary>
-        /// Creates a new void constant.
+        /// Creates a new void/uninitialized constant that is not a method parameter.
         /// </summary>
         public static ConstantValue Void()
         {
             return new ConstantValue(ConstantType.Void, 0);
         }
 
+        /// <summary>
+        /// Creates a new constant that signifies a method parameter.
+        /// </summary>
+        public static ConstantValue Parameter()
```

Comment:
This is unused due to the later change in local initialization, but keeping this for future use. This is a good distinction to make.